pkgs/test_api/lib/src/backend/compiler.dart

```diff
@@ -3,7 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 /// All the Dart compilers supported by the test runner.
-class Compiler {
+final class Compiler {
```

Comment:
Sure, I guess that's equally as breaking as adding `final` would be.

Which reminds me to double check - we should definitely skip the major version bump for `test`, are you OK with also skipping it for `test_core` and `test_api`? Some of these are technically breaking changes but since they don't impact google3 I think they are unlikely to impact anyone.

Reply:
> I assume that is implied if they don't break google3?

Yes, we'd see flutter usage in google3.

Reply:
sgtm then